cfe_internal/enterprise/federation/federation.cf

```diff
@@ -627,12 +653,13 @@ bundle agent superhub_schema
 {
   commands:
     am_superhub::
-      "$(cfengine_enterprise_federation:config.bin_dir)/psql_wrapper.sh"
+      "$(cfengine_enterprise_federation:config.bin_dir)/psql_wrapper.sh" -> { "CFE-951" }
         arglist => {
                      "cfdb",
                      `"select superhub_schema('$(sys.key_digest)');"`,
                    },
-        classes => psql_wrapper_exit_codes;
+        classes => psql_wrapper_exit_codes,
+        contain => default:in_dir( "/tmp" );
```

Comment:
no need for `/tmp`?

Reply:
Well, inside psql_wrapper.sh.mutache I see `su - cfpostgres`.

```
OUT=$(su - cfpostgres --command "{{{vars.sys.bindir}}}/psql --quiet --tuples-only --no-align --no-psqlrc \"$1\" --command=\"$2\"" 2> $TMP)
```

Reply:
I would say DRY tells us we should `cd /tmp` inside the psql_wrapper.sh.mustache file and leave it out of policy. But it's only two calls from policy anyway so doesn't get us that much. :)

Reply:
![image](https://user-images.githubusercontent.com/202896/158444985-4c0fffb7-8459-46dd-bc73-43a012e797b8.png)

Reply:
ACK.